students/esokada/lesson08/test_circle.py

```diff
@@ -0,0 +1,74 @@
+from circle import *
```

Comment:
avoid using `*` (star) imports; instead `from circle import Circle`

Reply:
Right! I'll be more careful in the future.